.travis.yml

```diff
@@ -73,6 +73,9 @@ install:
       cargo install hyperfine
     fi
     hyperfine --version
+    if [ ! $(which ab) ]; then
+      sudo apt-get install apache2-utils
```

Comment:
you could also just use a third party apt repo, or just compile wrk locally and put the binary on some server (github?) and just curl it down instead of recompiling it for every PR (this day and age's "don't print this paper" problem?)

Reply:
Ok, including wrk binaries in deno_third_party - 5.8M for mac and linux - seems reasonable. Patch updated.